.gitmodules

```diff
@@ -1,3 +1,6 @@
+[submodule "external/debugger-libs"]
+	path = external/debugger-libs
```

Comment:
This file should use 4 space indents, lest we get extra "line noise" during the next branch.